pkg/cloud/services/loadbalancer/loadbalancer_test.go

```diff
@@ -38,9 +41,24 @@ func Test_ReconcileLoadBalancer(t *testing.T) {
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
+	// Stub the call to net.LookupHost
+	lookupHost = func(host string) (addrs []string, err error) {
+		if net.ParseIP(host) != nil {
```

Comment:
I wonder if we should do this specifically in the non-test function.

Reply:
I don't think we need as I pointed out above.